tools/releasing/create_binary_release.sh

```diff
@@ -17,6 +17,9 @@
 # limitations under the License.
 #
 
+## check required Java and Maven version
```

Comment:
Yes, I [provided the code above](https://github.com/apache/flink/pull/23544#discussion_r1371232099) in case you still want to add this kind of check. It, at least, reduces the number of code redundancies (in terms of the Maven version) in comparison to the initial version of this PR. But [my comment above](https://github.com/apache/flink/pull/23544#discussion_r1371235162) still stands that the check itself is not really necessary for the stated reasons.

Reply:
gotcha, I will close this PR.